src/components/HomePageComponents/Section/Section.jsx

```diff
@@ -2,16 +2,21 @@ import React from "react";
 import "./Section.scss";
 
 const Section = (props) => {
-  const { title, bgColour, img, name } = props;
+  const { title, bgColour, img, name, type, padding } = props;
 
   return (
     <section className={`section ${bgColour}`} data-testid="section">
-      <h3 className="section_title">{title}</h3>
-      <p className="section__p">
-        Lorem ipsum dolor sit amet, consectetur adipisicing elit. Repellat
-        excepturi omnis laborum ullam libero quasi.
+      <h3 className="section__title">{title}</h3>
+
+      <p className={`section__p ${type}`}>
+        Convert and hold 54 currencies. Hold multiple currencies, and get the
+        real exchange rate when you convert. Compare our pricing We charge as
```

Comment:
```suggestion
        real exchange rate when you convert. Compare our pricing. We charge as
```

There is a . missing